src/Recommender/Service/RecommendedUsers.jsx

```diff
@@ -59,82 +59,101 @@ const RecommendedUsers = styled.div`
   padding-bottom: ${props.sidebar ? "12px" : "50px"};
 `;
 
+const NotEnoughData = styled.div`
+  width: 100vw;
```

Comment:
Sorry I didn't catch this earlier in my review. We should probably be using `width: 100%` to fill the container instead of `100vw`. This component is used in the `ActivityPage` sidebar and using `100vw` would break our layout (it would make the sidebar as wide as the screen).

Reply:
👍